Sphinx build failing

The recent build had failed after sphinx had been updated to 2.1.0, 
https://travis-ci.org/sympy/sympy/builds/540395440
I've confirmed that the fail is due to the update today.

So the duplicate references are in:

1. doc/src/modules/holonomic/operations.rst
2. doc/src/modules/holonomic/convert.rst

They both have references to
```
.. module:: sympy.holonomic.holonomic
```
I've tried deleting each and either way I get an error e.g.:
```
don't know which module to import for autodocumenting 'HolonomicFunction.integrate' (try placing a "module" or "currentmodule" directive in the document, or giving an explicit module name)
make: *** [html] Error 2
```

I think that `..module::` directive was used for mananging the context, allowing usage like `HolonomicFunction.integrate` instead of `sympy.holonomic.holonomic.HolonomicFunction.integrate`, so without the directive, it should be latter.

However, I think that it still fails with
```
duplicate object description of sympy.simplify.cse_main, other instance in modules/rewriting, use :noindex: for one of them
Makefile:49: recipe for target 'html' failed
```
and it seems like `module` directive is used in more places than holonomic.

I'm still not sure if sympy had been using deprecated features, or they had introduced some regressions.
If no good solution comes out in days, I'd support pinning down.

--- a/doc/src/modules/holonomic/convert.rst
+++ b/doc/src/modules/holonomic/convert.rst
@@ -1,7 +1,7 @@
 Converting other representations to holonomic
 =============================================
 
-.. module:: sympy.holonomic.holonomic
+.. currentmodule:: sympy.holonomic.holonomic
 
 Converting hypergeometric functions
 -----------------------------------

--- a/doc/src/modules/holonomic/operations.rst
+++ b/doc/src/modules/holonomic/operations.rst
@@ -29,7 +29,7 @@ a holonomic functions.
     >>> p * q
     HolonomicFunction((2) + (-2)*Dx + (1)*Dx**2, x, 0, [0, 1])
 
-.. module:: sympy.holonomic.holonomic
+.. currentmodule:: sympy.holonomic.holonomic
 
 Integration and Differentiation
 -------------------------------

--- a/doc/src/modules/holonomic/represent.rst
+++ b/doc/src/modules/holonomic/represent.rst
@@ -1,7 +1,7 @@
 Representation of holonomic functions in SymPy
 ==============================================
 
-.. module:: sympy.holonomic.holonomic
+.. currentmodule:: sympy.holonomic.holonomic
 
 Class :class:`DifferentialOperator` is used to represent the annihilator
 but we create differential operators easily using the function

--- a/doc/src/modules/holonomic/uses.rst
+++ b/doc/src/modules/holonomic/uses.rst
@@ -1,4 +1,4 @@
-.. module:: sympy.holonomic.holonomic
+.. currentmodule:: sympy.holonomic.holonomic
 
 Uses and Current limitations
 ============================

--- a/doc/src/modules/rewriting.rst
+++ b/doc/src/modules/rewriting.rst
@@ -61,7 +61,7 @@ use ``rewrite()`` to do specific type of rewrite::
 Common Subexpression Detection and Collection
 ---------------------------------------------
 
-.. module:: sympy.simplify.cse_main
+.. currentmodule:: sympy.simplify.cse_main
 
 Before evaluating a large expression, it is often useful to identify common
 subexpressions, collect them and evaluate them at once. This is implemented

